src/main/java/cli/Scrape.java

```diff
@@ -35,7 +33,7 @@ public void run() {
            header = "Scrape the PeopleSoft Class Search",
            description = "Scrape the PeopleSoft Class Search for a term")
   public void
-  term(@Mixin Mixins.Term termMixin, @Mixin Mixins.OutputFile outputFileMixin) {
+  term(@Mixin Mixins.TermOption termMixin, @Mixin Mixins.OutputFile outputFileMixin) {
```

Comment:
Using TermOption instead of Term is a great way to specify what could possibly be confusing (for example, ppl who just say QR)